avatar/avatar.css

```diff
@@ -0,0 +1,24 @@
+@import url(../style.css);
+
+.avatar {
+    height: 32px;
+    width: 32px;
+}
+
+.avatar-xl {
+    height: 48px;
```

Comment:
avoid using px as it might trouble you when doing responsiveness.

Reply:
okay, I'll take care of it. Thank you  !!